sgkit/curate.py

```diff
@@ -0,0 +1,75 @@
+from typing import Hashable
```

Comment:
Note sure about the name of this file. Another possibility might be `preprocessing.py`?

Reply:
Agreed, `preprocessing` is better